ParkingLotApi/Controllers/ParkingLotApiController.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using ParkingLotApi.DTO;
+using ParkingLotApi.Service;
+
+namespace ParkingLotApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ParkingLotApiController : ControllerBase
+    {
+        private readonly ParkingLotApiService parkingLotService;
+        public ParkingLotApiController(ParkingLotApiService parkingLotService)
+        {
+            this.parkingLotService = parkingLotService;
+        }
+
+        [HttpPost("ParkingLots")]
+        public async Task<ActionResult<ParkinglotDTO>> CreateParkingLot(ParkinglotDTO parkinglotDto)
+        {
+            if (parkinglotDto.Name == null || parkinglotDto.Capacity < 0 || parkinglotDto.Location == null
+            || parkinglotDto.Name == string.Empty || parkinglotDto.Location == string.Empty)
```

Comment:
good validation for empty input checkout

Reply:
extract a method